yccp/sweeps/transforms.py

```diff
@@ -207,6 +207,53 @@ def describe(self, paramset):
         )
 
 
+class ApplyFunctionElaborate(AddValue):
```

Comment:
Name does not describe its purpose.. Also, I do not see why this is needed. The function itself is given the `ParameterSet` as an argument and can extract the values in a straight-forward manner.

Or do you want to use a parameterized function (i.e., a closure)? This could also be created beforehand and then wrapped in `ApplyFunction`.